test/packages/good_content/manifest.yml

```diff
@@ -1,4 +1,4 @@
-format_version: 3.3.1
+format_version: 3.4.0
```

Comment:
Should it be updated the spec version for the compliance test here ? 

https://github.com/elastic/package-spec/blob/4366bf7327dc9d09b898e933797ccd1e7d091aef/compliance/features/basic.feature#L16-L19

That test is running successfully even if that spec version is not updated.

Reply:
If the package has SLOs it should be updated, yes.

Reply:
If that spec version is updated to be `@3.4.0`, I guess it should be needed to add a new compliance test for `3.4.0`.

Or should it be updated the one with stack `9.1.0-SNAPSHOT` to `3.4.0` ? @jsoriano 
https://github.com/elastic/package-spec/blob/4366bf7327dc9d09b898e933797ccd1e7d091aef/.buildkite/pipeline.trigger.compliance.tests.sh#L31

Would it be better to create a new content package without SLO , so it can be tested that scenario with previous spec versions `3.3.0` ?

Reply:
Added a new `basic_content` test package , to be able to keep testing content packages with spec version 3.3.0 (and previous Elastic stacks). Added in https://github.com/elastic/package-spec/pull/884/commits/6250bd13576aa15ff4ea2b8876ee836897c79ccd